pkg/client/cloudevents/grpcsource/watcherstore.go

```diff
@@ -137,7 +169,7 @@ func (m *RESTFulAPIWatcherStore) List(opts metav1.ListOptions) ([]*workv1.Manife
 }
 
 func (m *RESTFulAPIWatcherStore) ListAll() ([]*workv1.ManifestWork, error) {
```

Comment:
When is `ListAll()` called? 
I see here that it'll list all manifest works and I am trying to see and understand its behaviour

Reply:
`ListAll()` will not be called by this client, it just a interface implement

Reply:
Thanks for confirming